app/controllers/account/billing-info/invoices.js

```diff
@@ -0,0 +1,91 @@
+import Controller from '@ember/controller';
+import { filterBy } from '@ember/object/computed';
+import { computed } from '@ember/object';
+import EmberTableMixin from 'open-event-frontend/mixins/ember-table';
+	
+export default class extends Controller.extend(EmberTableMixin) {
+  // queryParams = ['page', 'per_page'];
+  // page = 1;
+  // per_page = 10;
+  // search = null;
+  // sort_dir = null;
+  // sort_by = null;
+  @filterBy('model.eventInvoices', 'status', 'due')
+  dueInvoices;
+
+  @filterBy('model.eventInvoices', 'status', 'paid')
+  paidInvoices;
+
+  @computed()
+  get dueInvoiceColumns() {
+    return [
+      {
+         name: 'Invoice ID',
+         valuePath: 'invoiceId'
+      },
+      {
+        name: 'Event Name',
+        valuePath: 'eventName'   
+      },
+      {
+        name: 'Date Issued',
+        valuePath: 'dateIssued'   
+      },
+      {
+        name: 'Amount',
+        valuePath: 'amount'   
+      },
+      {
+        name: 'View Invoice',
+        valuePath: 'invoiceLink'   
+      }
+
+    ];
+  }
+
+  @computed('dueInvoices')
+  get dueInvoiceRows() {
```

Comment:
Why is this needed ? Can't `dueInvoices` be used directly ?

Reply:
Did you mean that `dueInvoices` can be used directly instead of watching for them?

Reply:
Yes.